test/unit/helpers/people_helper_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class PersonHelperTest < ActionView::TestCase
+  include ApplicationHelper
+
+  test "it disambiguates names with trailing spaces" do
+    person_one = build(:person, forename: "Peter ", surname: "Jones")
```

Comment:
I'm probably being dense, but what's the significance of having 'Peter Jones' for person_one and person_two, but then stubbing them to return 'Vic' and 'Bob'? I'll stare at it a bit longer, probably all will become clear...

Great that we're adding a test for this!